fdbclient/Schemas.cpp

```diff
@@ -208,7 +208,10 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
                },
                "megabits_received":{
                   "hz":0.0
-               }
+               },
+							 "tls_policy_failures":{
```

Comment:
Looks like this file uses spaces.

Reply:
I think an analogous change needs to be made in documentation/sphinx/source/mr-status-json-schemas.rst.in. At one point there was effort to generate this automatically, but I'm not sure if it works well enough that we use it.